components/newsletterCallout.tsx

```
@@ -72,10 +72,12 @@ export default function NewsletterCallout({ newsletterData, globalFields }) {
     if (!email) {
       setIsSuccess(false);
       setError(new ToastableError("Email is required"));
+      return;
     }
     if (!selectedNewsLetters.length) {
       setIsSuccess(false);
       setError(new ToastableError("Please select at least one newsletter"));
+      return;
```

Maybe throw an error toast instead? User might not know why they're getting bonked

---

It's on the line above it

---

I'll see myself out 😅